_includes/configuration/dockerized-image-proxy.md

```diff
@@ -0,0 +1,92 @@
+### Deploy the TinyMCE image proxy server-side component using Docker (self-hosted enterprise bundle)
+
+The TinyMCE image proxy server-side component can be deployed using Docker or an container orchestration application such as Kubernetes. {{site.companyname}} provides a pre-configured package for creating a Image Proxy Docker image.
+
+> **Warning**: Do not push this docker image to a publicly accessible container registry. Doing so will constitute a breach of the [{{site.companyname}} Self-Hosted Software License Agreement](https://about.tiny.cloud/legal/tiny-self-hosted-software-license-agreement-enterprise/).
+
+#### Procedure
+
+The following procedure covers downloading, configuring, building and testing the {{site.productname}} Image Proxy Docker image.
+
+> **Note**: This procedure assumes that the **{{ site.productname }} Enterprise Bundle** has been extracted as described in [Download and extract the TinyMCE Enterprise Bundle](#downloadandextractthetinymceenterprisebundle).
+
+4. Navigate into the `tinymce-services` folder.
+
+    ```sh
+    cd tinymce-enterprise/tinymce-services/
+    ```
+
+5. Extract the contents of `ephox-hyperlinking-docker-starter-kit.zip`, such as:
+
+    ```sh
+    unzip ephox-hyperlinking-docker-starter-kit.zip -d hyperlinking-service-dockerfile
+    ```
+
+6. Copy `ephox-hyperlinking.war` into the directory containing the extracted files, such as:
+
+    ```sh
+    cp ephox-hyperlinking.war hyperlinking-service-dockerfile/
+    ```
+
+4. Navigate into the `hyperlinking-service-dockerfile` folder.
```

Comment:
4, 5, 6, 4, 5, 6 ... are you relying on Markdown's converter to fix these and it doesn't really matter what they are?

Reply:
per previous comment. I think a vscode plugin was trying to keep track of numbering